src/client/src/apps/MainRoute/widgets/footer-version/FooterVersion.tsx

```diff
@@ -6,7 +6,21 @@ export const Wrapper = styled.div`
   opacity: 0.59;
   font-size: 0.75rem;
 `
-
-const FooterVersion: FC = () => <Wrapper>{process.env.REACT_APP_BUILD_VERSION}</Wrapper>
+export const Link = styled.a`
+  color: inherit;
+  text-decoration: inherit;
+`
+const FooterVersion: FC = () => (
+  <Wrapper>
+    <Link
+      href={
+        'https://github.com/AdaptiveConsulting/ReactiveTraderCloud/releases/tag/' +
+        process.env.REACT_APP_VERSION
```

Comment:
@Mark-Adaptive mmm, I think we need a way to distinguish "non-tagged" versions from tagged ones, otherwise the URL will result in a 404.

e.g. v1.2.1 will exist, but not v1.2.1-13-43gfr33.

Not sure currently how to do this. Three options I can think of at the moment:
1. Pass another environment variable that indicates this.
2. Check if the version URL exists (i.e. doesn't lead to a 404) by calling an API/trying a fetch.
3. Make all links go to the last official version, i.e. extract just the first part of the version number (the v1.2.1 part), which should exist.

Of these, 2 seems the most reliable to me. Let's discuss on standup today. This is the GitHub REST API for querying tags: https://developer.github.com/v3/git/tags/. Since it's a public repo, that should work fine, but not sure how straightforward it is.

In any case - the logic should be:
1. retrieve version from env var
2. check whether version exists as a tag on github
2.1 if true, print version is clickable link
2.2 if false, just print the version

Reply:
I have been trying to implement solution 2 but I am receiving the following error when I try to fetch the link: 
"Access to fetch at 'https://github.com/AdaptiveConsulting/ReactiveTraderCloud/releases/tag/v1.5.2' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled." 
Do you happen to know any work arounds for this?

Reply:
Should have the proper behavior now: 
Clickable when version number does exist in the github releases:
![image](https://user-images.githubusercontent.com/66634974/98306794-8ad78b00-1f92-11eb-8813-7ac5dc892878.png)
Not Clickable when it doesn't exist: 
![image](https://user-images.githubusercontent.com/66634974/98306844-9d51c480-1f92-11eb-9db2-95bfea6d77de.png)